src/microbe_stage/organelle_unlocks/UndiscoveredOrganellesTooltip.cs

```diff
@@ -0,0 +1,97 @@
+﻿using Godot;
+
+/// <summary>
+///   The tooltip for undiscovered organelles
+/// </summary>
+public class UndiscoveredOrganellesTooltip : Control, ICustomToolTip
+{
+#pragma warning disable CA2213
```

Comment:
The dispose pragma disable *should not* be around the NodePaths as the style we use is to dispose those.